code/controllers/stonedmc/subsystem/job.dm

```diff
@@ -189,16 +189,16 @@ SUBSYSTEM_DEF(job)
 //Gives the player the stuff he should have with his rank
 /datum/controller/subsystem/job/proc/EquipRank(mob/M, rank, joined_late = FALSE)
 	var/mob/new_player/N
-	var/mob/living/H
```

Comment:
just why

Reply:
Because later into the file `H` is defined for human.